options.js

```diff
@@ -0,0 +1,83 @@
+const options = {
+  compatibilityMode: false,
+  copyButtons: true,
+  correctnewlines: false,
+  markdownOptions: {
+    fences: true,
+    commonmark: true,
+    gfm: true,
+    ruleSpaces: false,
+    listItemIndent: '1',
+    spacedTable: true,
+    paddedTable: true,
+    setext: true,
+  },
+  normalize: true,
+  settings: {
+    position: false,
+  },
+};
+
+// NOTE: disabling newline, paragraph, or text trips remark into an infinite loop!
```

Comment:
I can haz JSDOC comments?

```suggestion
/**
 * @note disabling `newline`, `paragraph`, or `text` tokenizers trips remark into an infinite loop!
 */
```